### PR TITLE
Remove unused methods and constants from legacy OC_API

### PR DESCRIPTION
**Pull request by MorrisJobke**
Ref #8375 and #7827

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
bildschirmfoto von 2018-03-12 18-09-42 :heart_eyes:

File: lib/private/legacy/api.php [hunk 1]
```diff
@@ -37,44 +37,10 @@
...
-
-	/** @deprecated Use \OCP\API::ADMIN_AUTH instead */
-	const ADMIN_AUTH = 3;
-
-	/**
-	 * API Response Codes
-	 */
-
-	/** @deprecated Use \OCP\API::RESPOND_UNAUTHORISED instead */
-	const RESPOND_UNAUTHORISED = 997;
```
Comment (rullzer) [inline lib/private/legacy/api.php]: Those are still used I believe... we probably should migrate

Comment (MorrisJobke) [reply lib/private/legacy/api.php]: I checked all of the apps within this organization and all of the App Store - they are only used in the ocdownloader (see link above) and the others use the ones from the public interface ;)

**Review by skjnldsv — APPROVED** [review]

File: lib/private/legacy/api.php (hunk 1 above)
Comment (rullzer) [reply lib/private/legacy/api.php]: Ah we have them double defined... but of course ...

**Review by rullzer — APPROVED** [review]

**Pull request merged**